xlml/apis/test_config.py

```diff
@@ -183,6 +185,22 @@ def setup_script(self) -> Optional[str]:
   def test_script(self) -> str:
     return '\n'.join(self.run_model_cmds)
 
+  @property
+  def startup_script(self) -> str:
+    if self.use_startup_script == False:
+      return None
+
+    main_command = '\n'.join(self.set_up_cmds + self.run_model_cmds)
+    final_command = f"""
+({main_command}) > /tmp/logs 2>&1 &
```

Comment:
Could we `| tee /tmp/logs` instead of a redirect here? It could be useful for debugging to have the logs stream to GCE through the startup-script output as well.

Reply:
I can do this, but when use `startup script`, we won't be able to see the logs stream unless we stream the logs through cloud logging like what `multihost_job` did (which could be a feature I can add in the future, but it is also strange to check logs outside of airflow. So I just print out the log from the 1st worker in `check_if_startup_script_end`)

Reply:
They do stream to cloud logging by default (I think through through the syslog, but would need to double check), it's just not tagged like `multihost_job`

Reply:
Talked to Ran offline, Airflow doesn't show logs from all workers. 

I will try to add cloud monitoring support in the next PR. it should be similar to `mulithost_job`

https://github.com/google/maxtext/blob/main/multihost_job.py#L200-L241